### PR TITLE
[2][Chore] Setup CI pipeline for unit test and static code linter

### LINKED ISSUE #2: [Chore] Setup CI pipeline for unit test and static code linter
## Why

- CI service is required

## Acceptance Criteria

- CI pipeline is run on Github Action
- Pipeline run unit test
- Pipeline run static code linter

### PR DESCRIPTION
**Pull request by chiiida**
https://github.com/llleyelll/ic-surveys-ios/issues/2

## What happened 👀

- Add fastlane scan into Github Action workflow
- Add SwiftLint into Github Action workflow
 
## Proof Of Work 📹

Screen Shot 2021-06-07 at 7 01 16 PM

### CODE REVIEW

**Review by markgravity — APPROVED** [review]
LGTM 🤖

**Comment by markgravity** [comment]
@llleyelll you may want to add this action the cancel previous runs 😁 
```
    - name: Cancel Previous Runs
       uses: styfle/cancel-workflow-action@0.5.0
       with:
         access_token: ${{ github.token }}
```

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  push:
+  pull_request:
+    types: [ opened, synchronize, edited, reopened ]
+
+jobs:
+  test:
+    name: Linter and Unit test
+    runs-on: macos-latest
+    steps:
+      - name: Git Checkout
```
Comment (markgravity) [inline .github/workflows/test.yml:18]: you may want to add this action the cancel previous runs 😁 
```
    - name: Cancel Previous Runs
       uses: styfle/cancel-workflow-action@0.5.0
       with:
         access_token: ${{ github.token }}
```

Comment (chiiida) [reply .github/workflows/test.yml:18]: Thank you. I'll make that change 😊

**Review by edgarss — APPROVED** [review]
LGTM

**Pull request merged**